pkg/services/hivelocity/hvtag/hvtag.go

```diff
@@ -135,6 +139,16 @@ func PermanentErrorTagFromList(tagList []string) (DeviceTag, error) {
 	return DeviceTagFromList(DeviceTagKeyPermanentError, tagList)
 }
 
+// DeviceUsableByCAPI returns if cluster can use the device.
+func DeviceUsableByCAPI(tagList []string) (bool, error) {
```

Comment:
this function should return a boolean and no error. The question is just yes or no. If something is wrong, then it's a no.